library/src/main/java/com/schibsted/spain/barista/interaction/BaristaClickInteractions.kt

```diff
@@ -26,72 +28,84 @@ import org.hamcrest.Matchers.anyOf
 
 object BaristaClickInteractions {
 
-    @JvmStatic
-    fun clickBack() {
-        pressBack()
-    }
+  @JvmStatic
+  fun clickBack() {
+    pressBack()
+  }
 
-    @JvmStatic
-    fun clickOn(resId: Int) {
-        performClickTypeOnMatcher(resId.resourceMatcher(), click())
-    }
+  @JvmStatic
+  fun clickOn(resId: Int) {
+    performClickTypeOnMatcher(resId.resourceMatcher(), click())
+  }
 
-    @JvmStatic
-    fun clickOn(text: String) {
-        performClickTypeOnMatcher(withText(text), click())
-    }
+  @JvmStatic
+  fun clickOn(text: String) {
+    performClickTypeOnMatcher(withText(text), click())
+  }
 
-    @JvmStatic
-    fun longClickOn(resId: Int) {
-        performClickTypeOnMatcher(resId.resourceMatcher(), longClick())
-    }
+  @JvmStatic
+  fun clickOn(@IdRes id: Int, text: String) {
+    performClickTypeOnMatcher(allOf(withId(id), withText(text)), click())
+  }
 
-    @JvmStatic
-    fun longClickOn(text: String) {
-        performClickTypeOnMatcher(withText(text), longClick())
-    }
+  @JvmStatic
+  fun longClickOn(resId: Int) {
```

Comment:
Should be `@IdRes id: Int` just to normalize the parameters name

Reply:
Goto https://github.com/SchibstedSpain/Barista/issues/158 😇